apps/dashboard/src/app/(app)/team/[team_slug]/[project_slug]/(sidebar)/webhooks/components/webhook-configs-table.tsx

```diff
@@ -0,0 +1,406 @@
+        teamSlug={props.teamSlug}
+        topics={props.topics}
+      />
+
+      {editingWebhook && (
+        <EditWebhookConfigModal
+          onOpenChange={(open) => {
+            if (!open) setEditingWebhook(null);
+          }}
+          open={!!editingWebhook}
+          projectSlug={props.projectSlug}
+          teamSlug={props.teamSlug}
+          topics={props.topics}
+          webhookConfig={editingWebhook}
+        />
+      )}
+
+      <DeleteWebhookModal
+        isPending={deleteMutation.isPending}
+        onConfirm={() => {
+          if (deletingWebhook) {
+            deleteMutation.mutate(deletingWebhook.id);
+          }
+        }}
+        onOpenChange={(open) => {
+          if (!open) setDeletingWebhook(null);
+        }}
+        open={!!deletingWebhook}
+        projectSlug={props.projectSlug}
+        teamSlug={props.teamSlug}
+        webhookConfig={deletingWebhook}
+      />
+    </div>
+  );
+}
+
+const sortByIcon: Record<SortById, React.FC<{ className?: string }>> = {
+  createdAt: CalendarIcon,
+  description: LetterTextIcon,
+  destinationUrl: WebhookIcon,
+  pausedAt: CheckIcon,
+};
+
+function SortDropdown(props: {
+  sortBy: SortById;
+  onSortChange: (value: SortById) => void;
+  hasActiveFilters: boolean;
+}) {
+  const values: SortById[] = [
+    "description",
+    "createdAt",
+    "destinationUrl",
+    "pausedAt",
+  ];
+  const valueToLabel: Record<SortById, string> = {
+    createdAt: "Creation Date",
+    description: "Description",
+    destinationUrl: "Destination URL",
+    pausedAt: "Requests",
+  };
+
+  return (
+    <DropdownMenu>
+      <DropdownMenuTrigger asChild>
+        <Button className="gap-1.5 rounded-full bg-card" variant="outline">
+          <ArrowUpDownIcon className="size-4 text-muted-foreground" />
+          <span className="hidden lg:inline">Sort by</span>
+        </Button>
+      </DropdownMenuTrigger>
+      <DropdownMenuContent
+        align="end"
+        className="w-60 rounded-xl p-1.5 shadow-lg"
+        sideOffset={10}
+      >
+        <DropdownMenuRadioGroup
+          className="flex flex-col gap-1"
+          onValueChange={(v) => props.onSortChange(v as SortById)}
+          value={props.sortBy}
+        >
+          {values.map((value) => {
+            const Icon = sortByIcon[value];
+            return (
+              <DropdownMenuItem
+                className={cn(
+                  "flex cursor-pointer items-center justify-between gap-2 rounded-lg py-2",
+                  props.sortBy === value && "bg-accent",
+                )}
+                key={value}
+                onClick={() => props.onSortChange(value)}
+              >
+                <div className="flex items-center gap-2">
+                  <Icon className="size-4 text-muted-foreground" />
+                  {valueToLabel[value]}
+                </div>
+
+                {props.sortBy === value ? (
+                  <CheckIcon className="size-4 text-foreground" />
```

Comment:
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/yNOf1svJ8o3zjO7zQouZ/42093605-922c-484d-b8b1-e97105957f07.png)

The icons for "Requests" and "Destination URL" is a bit weird